pandas/tseries/index.py

```diff
@@ -827,16 +828,21 @@ def _format_native_types(self, na_rep=u('NaT'),
     def to_datetime(self, dayfirst=False):
         return self.copy()
 
-    def astype(self, dtype):
+    def astype(self, dtype, copy=True):
         dtype = np.dtype(dtype)
```

Comment:
This and also a separate docstring and a mention in Other Enhancements in whatsnew doc?